transform/mattermost-analytics/models/marts/sales/fct_subscription_history.sql

```diff
@@ -1,11 +1,12 @@
-with subscription_history as (
-    select 
-        subscription_history_event_id
-        ,  subscription_id
-        ,  licensed_seats
-        , created_at
-
-    from {{ ref('stg_cws__subscription_history') }}
-)
-
-select * from subscription_history
\ No newline at end of file
+select 
+    sh.subscription_history_event_id
+    , sh.subscription_id
+    , s.customer_id
+    , COALESCE(sh.licensed_seats, s.QUANTITY) as licensed_seats
+    , sh.created_at
+    , s.CWS_DNS,
```

Comment:
```suggestion
    , s.cws_dns
```

Extra `,` at the end of line. Also since column name is not case sensitive, let's stick to lowercase to be consistent with the rest of the code.